src/nav/config/config_helper.py

```
@@ -0,0 +1,141 @@
+    if backplane_results_root_str is None:
+        try:
+            backplane_results_root_str = config.environment.backplane_results_root
+        except AttributeError:
+            pass
+    if backplane_results_root_str is None:
+        backplane_results_root_str = os.getenv('NAV_BACKPLANE_RESULTS_ROOT')
+    if backplane_results_root_str is None:
+        raise ValueError('One of --backplane-results-root, the configuration variable '
+                         '"environment.backplane_results_root", or the NAV_BACKPLANE_RESULTS_ROOT '
+                         'environment variable must be set'
+                         )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)